openstack-windows-winrm-blueprint.yaml

```diff
@@ -14,15 +14,29 @@ inputs:
     description: The HTTP web server port
     default: 8080
   agent_user:
-    description: User name used when SSH-ing into the started machine
+    description: User name used to remotely conntect to the machine via WinRM
   image:
     description: Openstack image name or id to use for the new server
   flavor:
     description: Openstack flavor name or id to use for the new server
+  network_name:
+    description: Openstack network name the new server will be connected to
+  floating_network_id:
+    description: The id of the network to use for allocating a floating ip
+  key_pair_name:
+    description: Openstack key pair name of the key to associate with the new server
+  private_key_path:
```

Comment:
I really don't like using keys. BTW if you use `install_method: init_script`, you don't even need SSH key.

Reply:
Well, this is specifically a WinRM example, so init_script isn't applicable here (we do have another blueprint for that).

Reply:
Ah right. Do! Well, in any that case, we still support agent installation via user data. (Or we are supposed to at least.)